modules/spring-data-2.0-ext/pom.xml

```diff
@@ -39,12 +39,14 @@
             <groupId>org.apache.ignite</groupId>
             <artifactId>ignite-core</artifactId>
             <version>${ignite.version}</version>
+            <scope>provided</scope>
```

Comment:
Hi Mikhail, My understanding is we will always depend on latest snapshot of master from Ignite Nightly build in Apache Ignite Extensions project.

Here is the discussion thread on the same http://apache-ignite-developers.2346864.n4.nabble.com/DISCUSS-dependencies-and-release-process-for-Ignite-Extensions-td44478.html

 When we release specific extension like spring-data module then during release we will set latest release of Ignite version in the release branch for release. In this case it would be 2.9.0 version.